db/seeds.rb

```diff
@@ -190,6 +190,7 @@
     username: "Admin_McAdmin",
     profile_image: File.open(Rails.root.join("app/assets/images/#{rand(1..40)}.png")),
     confirmed_at: Time.current,
+    registered_at: Time.current,
```

Comment:
Great catch. I see when we added this the other users were updated (and the registrations controller does this automatically for new user sign ups). A backfill script was added to set all existing users' times to their created at time https://github.com/forem/forem/pull/11008/files#diff-2b78ee666a74f9a39c9c5c19d103070b9d216fb83a1ca52e86cf073cc28d8156R6

I'm not opposed to adding a validation (which would merit a test) but this change is fine as-is.

One valid reason not to add a validation is that invited users (invitation sent, not accepted) will exist but not have a registration time associated (since they've never logged in).

Reply:
also - I can confirm the error was happening for me on main (I got NoMethodError sending > to nil), and this PR's branch fixed it.